vite_ruby/lib/vite_ruby/build.rb

```diff
@@ -4,7 +4,7 @@
 require 'time'
 
 # Internal: Value object with information about the last build.
-ViteRuby::Build = Struct.new(:success, :timestamp, :vite_ruby, :digest, :current_digest, :last_build_path) do
+ViteRuby::Build = Struct.new(:success, :timestamp, :vite_ruby, :digest, :current_digest, :last_build_path, :err_msg) do
```

Comment:
At some point I'd like to drop Ruby 2.4, and switch this to `keyword_init`.

Even for an internal structure, the amount of parameters makes initializing this very error prone.

Reply:
Yes, I definitely had that thought too. Ruby 2.4 went EOL on 2020-03-31

Reply:
You could also stop using the convenience Struct if you really wanted kw args plus ruby 2.4, at the cost of more boilerplate code.